Dribbblish-dynamic/README.md

```diff
@@ -0,0 +1,84 @@
+# Dribbblish-dynamic
+
```

Comment:
Add explanation of main difference from Dribbblish

Reply:
Renamed to DribbblishDynamic and explained the main difference